source/guide_phpbb.rst

```diff
@@ -0,0 +1,280 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: board
+.. tag:: forum
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/phpbb.svg
+      :align: center
+
+#####
+phpBB
+#####
+
+.. tag_list::
+
+phpBB_ is a free flat-forum bulletin board software solution that can be used to stay in touch with a group of people.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+  * :manual:`mail <mail-access>`
+
+License
+=======
+
+phpBB_ is released under the `GPLv2 License`_.
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your phpBB board with your own domain, you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Create Database
+===============
+
+phpBB saves your data in a :manual:`MySQL <database-mysql>` database. It is recommended to use an :manual_anchor:`additional database <database-mysql.html#additional-databases>` (e.g. ``isabell_phpbb``) instead of the default database.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mysql --verbose --execute="CREATE DATABASE ${USER}_phpbb"
+ --------------
+ CREATE DATABASE isabell_phpbb
+ --------------
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+
+Go to the phpBB download_ website and copy the URL of the latest ZIP release, e.g. ``https://download.phpbb.com/pub/release/3.3/3.3.3/phpBB-3.3.3.zip``.
+
+Then, ``cd`` to your :manual:`document root <web-documentroot>` and download the latest release of phpBB and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd ~/html/
+ [isabell@stardust html]$ curl -L https://download.phpbb.com/pub/release/3.3/3.3.3/phpBB-3.3.3.zip -o phpbb.zip
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  100 25565    0 25565    0     0  22942      0 --:--:--  0:00:01 --:--:-- 22948
+  100 5138k  100 5138k    0     0   769k      0  0:00:06  0:00:06 --:--:-- 1108k
+ [isabell@stardust html]$ unzip phpbb.zip
+ [isabell@stardust html]$ mv phpBB3/* .
```

Comment:
```
[phpbb@ursa html]$ ll -a phpBB3/
total 8
drwxr-xr-x.  2 phpbb phpbb   23 Apr  5 21:33 .
drwxr-xr-x. 20 phpbb phpbb 4096 Apr  5 21:33 ..
-rw-r--r--.  1 phpbb phpbb 2734 Jan 25 22:07 .htaccess
```
This leaves the `.htaccess` behind. What do you think about this?
```
curl -L https://download.phpbb.com/pub/release/3.3/3.3.3/phpBB-3.3.3.tar.bz2 -o phpbb.tar.bz2
tar --strip-components 1 -xf phpbb.tar.bz2
rm phpbb.tar.bz2
```

Reply:
Thanks, I will also move the htaccess file.

Reply:
Is there something wrong with the tar version? :sweat_smile: it saves two commands (additional `mv` and `rm`), which usually reduces the number of errors made, when people use the guide.

Reply:
No, of course there is nothing wrong with it :) It's a long day for me, I just have overseen it. Commit will come.